src/jwt.ts

```diff
@@ -0,0 +1,11 @@
+export function parseExp (token: string): number {
+  const payloadBase64Url = token.split('.')[1]
+  const payloadBase64 = payloadBase64Url.replace(/-/g, '+').replace(/_/g, '/')
+  const jsonPayload = decodeURIComponent(
+    atob(payloadBase64)
+      .split('')
+      .map((c) => '%' + ('00' + c.charCodeAt(0).toString(16)).slice(-2))
+      .join('')
+  )
+  return JSON.parse(jsonPayload).exp
+}
```

Comment:
I would prefer only a function `peekPayload`, grabbing just the `exp` from it seems unnecessary